docs/usage.md

```diff
@@ -35,3 +35,54 @@ methods: {
   }
 }
 ```
+
+### Cancel token
+
+You can cancel a request using a *cancel token*.
+
+> The axios cancel token API is based on the withdrawn [cancelable promises proposal](https://github.com/tc39/proposal-cancelable-promises).
+
+You can create a cancel token using the `CancelToken.source` factory as shown below:
+
+```js
+const CancelToken = this.$axios.CancelToken;
+const source = CancelToken.source();
+
+this.$axios.$get('/user/12345', {
+  cancelToken: source.token
+}).catch(function (thrown) {
+  if (this.$axios.isCancel(thrown)) {
+    console.log('Request canceled', thrown.message);
+  } else {
+    // handle error
+  }
+});
+
+this.$axios.$post('/user/12345', {
+  name: 'new name'
+}, {
+  cancelToken: source.token
+})
+
+// cancel the request (the message parameter is optional)
+source.cancel('Operation canceled by the user.');
+```
+
+You can also create a cancel token by passing an executor function to the `CancelToken` constructor:
+
+```js
+const CancelToken = this.$axios.CancelToken;
+let cancel;
+
+this.$axios.$get('/user/12345', {
+  cancelToken: new CancelToken(function executor(c) {
```

Comment:
Can't we simply instantiate CancelToken outside?

Reply:
These are the examples in the axios docs. 

Personally I'd leave it as is and state that you're quoting rather than treating these doc as part of this project.